metricbeat/module/haproxy/info/data.go

```diff
@@ -145,14 +145,16 @@ func eventMapping(info *haproxy.Info) (common.MapStr, error) {
 			// Convert this value to a float between 0.0 and 1.0
 			fval, err := strconv.ParseFloat(f.Interface().(string), 64)
```

Comment:
I was thinking to open an issue to _improve_ potential nil errors here. WDYT @ruflin?

Reply:
sure. If it's a quick fix, no need for an issue, directly to a PR ;-)